services/secondary-storage/server/src/main/java/org/apache/cloudstack/storage/resource/NfsSecondaryStorageResource.java

```diff
@@ -1139,7 +1139,7 @@ protected File downloadFromUrlToNfs(String url, NfsTO nfs, String path, String n
             }
             return new File(destFile.getAbsolutePath());
         } catch (IOException e) {
-            s_logger.debug("Faild to get url:" + url + ", due to " + e.toString());
+            s_logger.debug("Failed to get url:" + url + ", due to " + e.toString());
```

Comment:
LGTM, just a small suggestion:

```suggestion
            s_logger.debug("Failed to get url: " + url + ", due to " + e.toString());
```